tdrs-backend/tdpservice/users/migrations/0012_add_initial_super_user.py

```diff
@@ -0,0 +1,44 @@
+import os
+from django.contrib.auth.hashers import make_password
+from django.db import migrations
+from django.utils import timezone
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('users', '0011_auto_20210108_1741'),
+    ]
+
+    def generate_superuser(apps, schema_editor):
+        # Use the historical model to prevent this from failing on clean
+        # builds if the User model changes in the future
+        User = apps.get_model('users', 'User')
+
+        # set the environment variable to the username of the
+        # initial superuser
+        su_username = os.environ.get('DJANGO_SU_NAME', 'admin')
```

Comment:
Some things to think about here
1. In the case of the dev server, this user will likely already exist, so we should first check to see if the user is already there and update that user, if so
2. We should probably throw an exception if `DJANGO_SU_NAME` is blank or null